pkg/controllers/events/cluster_eventmapper.go

```diff
@@ -65,14 +56,14 @@ func (m *ClusterEventMapper) mapToPolicyRequest(ctx context.Context, obj client.
 
 	requests := make([]reconcile.Request, 0)
 	for _, gw := range allGwList.Items {
-		val := metadata.GetAnnotation(&gw, gateway.GatewayClustersAnnotation)
+		val := metadata.GetAnnotation(&gw, "kuadrant.io/gateway-clusters")
```

Comment:
wondering why we moved away from a constant?

Reply:
I think the idea Mike had while doing it - remove as many ties to the gateway package as possible to ease the transition to a separate repo :thinking:  could undo it, if you prefer.

Reply:
This was more because the constant is in the gateway package and that now imports this. I think I'll event handler back into the gateway package since its only relevant there.